apps/accounts/views.py

```diff
@@ -49,3 +65,129 @@ def get_auth_token(request):
 
     response_data = {"token": "{}".format(token)}
     return Response(response_data, status=status.HTTP_200_OK)
+
+
+@api_view(["POST"])
+@throttle_classes([UserRateThrottle])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication,))
+def invite_users_to_challenge(request, challenge_pk):
+
+    challenge = get_challenge_model(challenge_pk)
+
+    try:
+        challenge_host = ChallengeHost.objects.get(user=request.user.pk)
+    except ChallengeHost.DoesNotExist:
+        response_data = {"error": "You're not a challenge host"}
+        return Response(response_data, status=status.HTTP_400_BAD_REQUEST)
+
+    if not is_user_a_host_of_challenge(request.user, challenge.pk):
+        response_data = {
+            "error": "You're not authorized to invite a user in {}".format(
+                challenge.title
+            )
+        }
+        return Response(response_data, status=status.HTTP_400_BAD_REQUEST)
+
+    users_email = request.data.get("emails")
+    subject = request.data.get("subject")
```

Comment:
Let's template the email title at our end. Something like 'You have been invited to {CHALLENGE_NAME} on EvalAI'